cmd/crc/cmd/root.go

```diff
@@ -96,15 +98,26 @@ func setProxyDefaults() {
 	httpProxy := config.GetString(cmdConfig.HTTPProxy.Name)
 	httpsProxy := config.GetString(cmdConfig.HTTPSProxy.Name)
 	noProxy := config.GetString(cmdConfig.NoProxy.Name)
+	proxyCAFile := config.GetString(cmdConfig.ProxyCAFile.Name)
+
+	// If proxyCAFile is not empty string then read the content and pass it to NewProxyDefaults
+	var proxyCACert []byte
+	if proxyCAFile != "" {
+		var err error
+		proxyCACert, err = ioutil.ReadFile(proxyCAFile)
+		if err != nil {
+			exit.WithMessage(1, fmt.Sprintf("not able to read proxyCAFile %s: %v", proxyCAFile, err.Error()))
+		}
+	}
```

Comment:
I'd go with a `getProxyCAData` helper rather than having this low-level code here, + the trim/conversion to string in the `NewProxyDefault` call.